### DISCUSSION TITLE
[Help!] My account was suspended!

### DISCUSSION THREAD

**Discussion by ruoyuTz** [community/community#117437] [New to GitHub]
Dear Github Administrator,

Hello! I am a newcomer to using Github. Two days ago, my account (username: TongZhao1030) was suspended, and as of now, it has not been resolved. I want to clarify that I am more than willing to comply with Github's policies. If I have violated any policies, it was unintentional (perhaps due to switching VPNs while using the platform). I am sincerely sorry for any inconvenience caused. I kindly request an opportunity to correct my mistake, and I assure you that I will not repeat it in the future. The code in my Github repository is not backed up, so it is extremely important to me. 
Thank you very much for your understanding. I have asked help through Github Support and my Ticket ID is 2713785.

Best regards,
Tong

**Comment by ruoyuTz** [comment]
Please help me and I really need my account back!!! 
Thanks a lot.

**Comment by ruoyuTz** [comment]
@dmilburn @queenofcorgis 
Hi there, I am sorry to bother you but it has been 4 days since my account was suspended. Could you please help me check my request?

Thank you very much!!!!!!

**Comment by ghostinhershell** [comment]
Hi @ruoyuTz,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
